docs/sphinx/conf.py

```diff
@@ -347,4 +347,4 @@
     linkcheck_timeout = 30
 
 # Regular expressions that match URIs that should not be checked when doing a linkcheck build
-linkcheck_ignore = ["https://www.leica-microsystems.com"]
+linkcheck_ignore = ["https://micro-manager.org/wiki/"]
```

Comment:
Issues from last Friday seem to have been resolved. Want to try again to remove from the ignore list?

Reply:
That's the 2nd or 3rd time that link has been flaky, I'm wondering if we should just leave it for now?